build/charts/flow-aggregator/templates/deployment.yaml

```diff
@@ -15,6 +15,14 @@ spec:
       labels:
         app: flow-aggregator
     spec:
+      {{- if .Values.hostAliases }}
+      hostAliases:
+      {{- range $v := .Values.hostAliases }}
+        - ip: {{ $v.ip }}
+          hostnames:
+            - {{ $v.hostname }}
+      {{- end }}
+      {{- end }}
```

Comment:
this should be:
```yaml
      hostAliases:
      {{- range .Values.hostAliases }}
        - ip: {{ .ip }}
          hostnames:
          {{- range $hostname := .hostnames }}
            - {{ $hostname }}
          {{- end }}
      {{- end }}
```

I think we want to support the exact same format as the PodSpec (so multiple hostnames)

I also don't think you need conditionals

Reply:
The IDE will display a error message, stating "value is required", if I don't add the if condition. But it has no impact when applying the yaml file.
<img width="483" alt="image" src="https://github.com/antrea-io/antrea/assets/59460118/8b00717e-929b-4a7b-a36e-86f4e5c371af">